plugins/ecs-bridge/engine/engine.go

```diff
@@ -73,7 +74,9 @@ func (engine *engine) CreateBridge(bridgeName string, mtu int) (*netlink.Bridge,
 	if bridge == nil {
 		err = engine.createBridge(bridgeName, mtu)
 		if err != nil {
-			return nil, err
+			// If createBridge returns error, still proceed to lookupBridge because the error
+			// may just indicate someone else already created the bridge
+			log.Warnf("Unable to create bridge: %v", err)
```

Comment:
I think a better place to log this error is in `commands.go`, [here](https://github.com/aws/amazon-ecs-cni-plugins/blob/0c6216c60401232805e50e31d4040ae84b0b23cf/plugins/ecs-bridge/commands/commands.go#L30). That way, we don't have to add a `log` statement every time an error is returned and instead log it in one place (actually fix the `TODO` [here](https://github.com/aws/amazon-ecs-cni-plugins/blob/0c6216c60401232805e50e31d4040ae84b0b23cf/plugins/ecs-bridge/commands/commands.go#L42). We can add additional context by wrapping this with `errors.Wrap()`. At the very least, we should be logging this in `commands.go`, [where it's being invoked](https://github.com/aws/amazon-ecs-cni-plugins/blob/0c6216c60401232805e50e31d4040ae84b0b23cf/plugins/ecs-bridge/commands/commands.go#L49).

Reply:
i added the log in commands.go and fix the TODO. i think wrapping the createBridge error together with the error from lookupBridge seems weird though, so still keeping the warning log here..